### DISCUSSION TITLE
Chnage the Font of the Current Session Using PowerShell

### DISCUSSION THREAD

**Discussion by RoyiAvital** [microsoft/terminal#16801] [Q&A] [answered]
Hi,

Is there a way, using `PowerShell`, to change the font used in the current session (Tab) of Windows Terminal?
I can set it per profile, I wonder if I can set it on the fly during the session.

**Comment by lhecker** [comment, marked as answer]
At the moment this is not possible, but you can subscribe to issue #6779 to be notified once that changes. We'd also be happy to accept a PR for this feature.
